zerver/lib/streams.py

```diff
@@ -312,22 +312,22 @@ def access_stream_for_send_message(
 
     # All other cases are an error.
     raise JsonableError(
-        _("Not authorized to send to stream '{channel_name}'").format(channel_name=stream.name)
+        _("Not authorized to send to channel '{channel_name}'").format(channel_name=stream.name)
     )
 
 
 def check_for_exactly_one_stream_arg(stream_id: Optional[int], stream: Optional[str]) -> None:
     if stream_id is None and stream is None:
-        raise JsonableError(_("Please supply 'stream'."))
+        raise JsonableError(_("Please supply a channel parameter: name or ID."))
 
     if stream_id is not None and stream is not None:
-        raise JsonableError(_("Please choose one: 'stream' or 'stream_id'."))
+        raise JsonableError(_("Please supply only one channel parameter: name or ID."))
 
 
 def check_stream_access_for_delete_or_update(
     user_profile: UserProfile, stream: Stream, sub: Optional[Subscription] = None
 ) -> None:
-    error = _("Invalid stream ID")
+    error = _("Invalid channel ID")
```

Comment:
A good refactor for this file might be to just make an `InvalidChannelIDError` class and raise those exceptions rather than doing `JsonableError`s. But it's not essential for this cutover project.